### PR TITLE
Added props to Router

### PR DESCRIPTION
**Pull request by Rohit-wednesday**
### Ticket Link
---------------------------------------------------

### Related Links
---------------------------------------------------

### Description
---------------------------------------------------
Added props to Router

### Steps to Reproduce / Test
---------------------------------------------------

### GIF's
---------------------------------------------------

### CODE REVIEW

File: app/containers/App/index.js [hunk 1]
```diff
@@ -27,14 +27,23 @@ function App({ location }) {
...
+            const Component = routeConfig[routeKey].component;
+            return (
+              <Route
+                exact={routeConfig[routeKey].exact}
+                key={index}
+                path={routeConfig[routeKey].route}
+                render={props => {
+                  const updatedProps = {
+                    ...props,
+                    ...routeConfig[routeKey].props
```
Comment (alichherawalla) [inline app/containers/App/index.js]: this will break when we spread over undefined

File: app/containers/App/index.js [hunk 2]
```diff
@@ -27,14 +27,23 @@ function App({ location }) {
     <ThemeProvider theme={theme}>
       <div>
         <Switch>
-          {_.map(Object.keys(routeConfig), (routeKey, index) => (
-            <Route
-              exact={routeConfig[routeKey].exact}
-              key={index}
-              path={location.pathname + routeConfig[routeKey].route}
-              component={routeConfig[routeKey].component}
```
Comment (alichherawalla) [inline app/containers/App/index.js]: also i don't see the changes in the routeConfig in your branch 
https://github.com/wednesday-solutions/react-template/blob/feature/add-props-to-router/app/routeConfig.js#L1

can you please update the routeConfig as well

File: app/containers/App/index.js (hunk 1 above)
Comment (Rohit-wednesday) [reply app/containers/App/index.js]: but the spread silently ignores undefined.

File: app/containers/App/index.js (hunk 2 above)
Comment (Rohit-wednesday) [reply app/containers/App/index.js]: will do sir

File: app/containers/App/index.js (hunk 1 above)
Comment (alichherawalla) [reply app/containers/App/index.js]: makes sense!

File: app/routeConfig.js [hunk 3]
```diff
@@ -5,6 +5,12 @@ export const routeConfig = {
   repos: {
     component: HomeContainer,
     route: '/',
+    // add props to pass to HomeContainer
+    props: {
+      maxwidth: 500,
+      padding: 20
+      // other_props: 'something_something'
```
Comment (alichherawalla) [inline app/routeConfig.js]: Remove this comment

File: app/containers/HomeContainer/index.js [hunk 4]
```diff
@@ -51,7 +51,8 @@ export function HomeContainer({
   reposData = {},
   reposError = null,
   repoName,
-  history
+  history,
+  ...props
 }) {
```
Comment (alichherawalla) [inline app/containers/HomeContainer/index.js]: Destructure maxWidth and padding here

**Pull request merged**